docs-src/themes/docdock/.gitignore

```diff
@@ -0,0 +1,10 @@
+.DS_Store
```

Comment:
Forgive my ignorance, do we need all of these `docs-src/themes/docdock/*` files?